### PR TITLE
tests: run spread tests on arm64 instances in google cloud

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Run the spread tests in arm64 servers

Bacause of fhis change, it is required to use a different backend for arm as those servers are just allowed in a different timezone. 

Also a new spread with some changes is needed because the arm servers don't allow to write to the serial port device (/dev/ttyS*).

The change also includes some changes to make tests work.

### CODE REVIEW

File: tests/main/lxd-mount-units/task.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ execute: |
         lxd.lxc exec ubuntu -- sh -c "echo https_proxy=$https_proxy >> /etc/environment"
     fi
     # wait for the container to be fully up
-    (lxd.lxc exec ubuntu -- systemctl --wait is-system-running || true) | MATCH "(running|degraded)"
+    retry --wait 1 -n 10 sh -c "lxd.lxc exec ubuntu -- systemctl --wait is-system-running | grep -Eq (running|degraded)"
```
Comment (mardy) [inline tests/main/lxd-mount-units/task.yaml:44]: This seems unrelated, I'd suggest moving into its own PR.

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -451,16 +453,23 @@ jobs:
           # Register a problem matcher to highlight spread failures
           echo "::add-matcher::.github/spread-problem-matcher.json"
 
+          BACKEND=google
+          SPREAD=spread
+          if [[ "${{ matrix.system }}" =~ -arm- ]]; then
+              BACKEND=google-arm
+              SPREAD=spread-arm
```
Comment (mardy) [inline .github/workflows/test.yaml:460]: It's not clear why we need a different spread command (and where it is).

**Review by mardy — APPROVED** [review]
Couple of remarks, otherwise LGTM!

File: tests/main/lxd-mount-units/task.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply tests/main/lxd-mount-units/task.yaml:44]: I added this line because I reproduced the same issue in arm as it is happening with the kenetic images.

File: .github/workflows/test.yaml (hunk 2 above)
Comment (sergiocazzolato) [reply .github/workflows/test.yaml:461]: well, this is something temporal until we have the change merged in spread, so I'll remove spread-arm
